ci/configure_hdf5_mac.sh

```diff
@@ -0,0 +1,49 @@
+function set_compiler_vars() {
+    arch=$1
+    export CC="/usr/bin/clang"
+    export CXX="/usr/bin/clang"
+    export CFLAGS="$CFLAGS -arch $arch"
+    export CPPFLAGS="$CPPFLAGS -arch $arch"
+    export CXXFLAGS="$CXXFLAGS -arch $arch"
+}
+
+
+function build_zlib() {
+    ZLIB_VERSION="1.2.11"
```

Comment:
```suggestion
    ZLIB_VERSION="1.2.12"
```

Looks like it was just released the other day. :slightly_smiling_face:

Reply:
Fixed